sql/catalyst/src/main/scala/org/apache/spark/sql/types/DataTypeConversions.scala

```diff
@@ -20,24 +20,21 @@ package org.apache.spark.sql.types
 import java.text.SimpleDateFormat
 
 import org.apache.spark.sql.Row
-import org.apache.spark.sql.catalyst.ScalaReflection
-import org.apache.spark.sql.catalyst.expressions.GenericMutableRow
+import org.apache.spark.sql.catalyst.CatalystTypeConverters
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
 
 
 private[sql] object DataTypeConversions {
 
   def productToRow(product: Product, schema: StructType): Row = {
```

Comment:
I don't see the change here yet -- but if no one is using this method I would be very happy to remove it.